### ISSUE TITLE
Unable to run JUnit tests

### ISSUE THREAD

**Issue by agasser** [wpilibsuite/vscode-wpilib#114]
I am unable to compile and run simple JUnit tests. The following error message prints at the console:
```
> Task :compileTestJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all files for configuration ':testCompileClasspath'.
> Could not find junit:junit:4.12.
  Searched in the following locations:
    - file:/C:/Users/Public/frc2019/maven/junit/junit/4.12/junit-4.12.pom
    - file:/C:/Users/Public/frc2019/maven/junit/junit/4.12/junit-4.12.jar
    - http://first.wpi.edu/FRC/roborio/maven/release/junit/junit/4.12/junit-4.12.pom
    - http://first.wpi.edu/FRC/roborio/maven/release/junit/junit/4.12/junit-4.12.jar
    - http://wpimirror.imjac.in/m2/release/junit/junit/4.12/junit-4.12.pom
    - http://wpimirror.imjac.in/m2/release/junit/junit/4.12/junit-4.12.jar
  Required by:
      project :

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 4s
3 actionable tasks: 2 executed, 1 up-to-date
The terminal process terminated with exit code: 1
```

Steps to reproduce:
1) I create a new project using the TimedRobot template.
2) Create the following folder structure under the `src` folder: `/test/java/frc/robot`
3) Create a new file called `MyTest.java`
4) In the new file, type "class" followed by CTRL + SPACE, then select the class template.
5) Open the WPI Lib command palette and select _WPILib: Test Robot Code_

Expected outcome
The test builds.

Actual outcome
The error message above prints at the console.

I also tried running the tests directly with the Gradle wrapper and the results was the same: `./gradlew test`

**Comment by agasser** [comment]
I forgot to mention the version. I am using WPILib Beta 1 from TeamForge.

**Comment by ThadHouse** [comment]
Whoops. We forgot to add maven Central to the build.gradle file. Add the stuff below to somewhere in your build.gradle file 

```
repositories {
    mavenCentral()
}
```

**Comment by ThadHouse** [comment]
Reminder to me, I need to include junit in the offline installer.

**Comment by agasser** [comment]
Thanks for the workaround @ThadHouse. It worked for me.

**Comment by ThadHouse** [comment]
The mavenCentral part of this has been fixed. The Gradle 5.0 issue is still in flight.

**Comment by ThadHouse** [comment]
This got closed because we now use 5.4 for gradle.
